src/main/java/net/starschema/clouddb/jdbc/BQStatementRoot.java

```diff
@@ -197,7 +198,7 @@ public boolean execute(String arg0, boolean unlimitedBillingBytes) throws SQLExc
      * @throws BQSQLFeatureNotSupportedException
      */
 
-    public boolean execute(String arg0, int arg1) throws SQLException {
+    public boolean execute(String sql, int autoGeneratedKeys) throws SQLException {
```

Comment:
Just added the names of the arguments so it's more legible according to the javadocs @ https://docs.oracle.com/javase/7/docs/api/java/sql/Statement.html